src/oasis-indexer/orval.config.js

```diff
@@ -3,10 +3,10 @@ import removeNetworkFromName from './removeNetworkFromName.js'
 
 /** @type {import('@orval/core').Config} */
 const config = {
-  indexer: {
+  nexus: {
     input: {
       // target: './v1.yaml',
-      // target: 'https://raw.githubusercontent.com/oasisprotocol/oasis-indexer/main/api/spec/v1.yaml',
+      // target: 'https://raw.githubusercontent.com/oasisprotocol/nexus/main/api/spec/v1.yaml',
       // target: 'https://index-staging.oasislabs.com/v1/spec/v1.yaml',
       target: 'https://index.oasislabs.com/v1/spec/v1.yaml',
```

Comment:
I assume the domain will be renamed at some point too, but for now, we need to stick with the old name.